Readme.md

```diff
@@ -59,20 +59,21 @@ describe('makePoniesPink', () => {
   beforeAll(() => {
     /* Runs before all tests */
   })
-  afterAll(() => {
-    /* Runs after all tests */
-  })
   beforeEach(() => {
     /* Runs before each test */
   })
-  afterEach(() => {
-    /* Runs after each test */
-  })
 
   test('make each pony pink', () => {
     const actual = fn(['Alice', 'Bob', 'Eve'])
     expect(actual).toEqual(['Pink Alice', 'Pink Bob', 'Pink Eve'])
   })
+  
+  afterEach(() => {
```

Comment:
This makes them invisible, let's move them back.

Reply:
Alright, I will